testdata/list_opcodes_auth_direct.test.yaml

```diff
@@ -0,0 +1,48 @@
+spec:
+  name: list_opcodes_auth_direct
+  generator: list_opcodes_auth_direct
+  description: List opcodes using direct authentication
+  request:
+    header:
+      magic_number: 1589683984
+      header_size: 30
+      major_version_number: 1
+      minor_version_number: 0
+      flags: 0
+      provider: 0
+      session_handle: 0
+      content_type: 0
+      accept_type: 0
+      auth_type: 0
+      content_length: auto
+      auth_length: auto
+      opcode: 9
+      status: 0
+    body_description: provider id 0x01
+    auth:
+      type: direct
+      app_name: jimbob
+  result:
+    header:
+      magic_number: 1589683984
+      header_size: 30
+      major_version_number: 1
+      minor_version_number: 0
+      flags: 0
+      provider: 0
+      session_handle: 0
+      content_type: 0
+      accept_type: 0
+      auth_type: 0
+      content_length: auto
+      auth_length: auto
+      opcode: 9
+      status: 0
+    body_description: list opcodes result [1,3,2]
+    auth:
+      type: direct
+      app_name: jimbob
+    expected_parse_result: succeed
+test_data:
+  request: EKfAXh4AAQAAAAAAAAAAAAAAAAAAAAIAAAAGAAkAAAAAAAAAamltYm9iCAE=
+  result: EKfAXh4AAQAAAAAAAAAAAAAAAAAAAAUAAAAGAAkAAAAAAAAAamltYm9iCgMBAwI=
```

Comment:
Could the `testdata/*` files only contain those two fields instead of the `spec` + those two fields? The `testdata/` files are only going to be read by the mock service right? The clients test writers could only read at the spec?